web/router.ex

```diff
@@ -17,6 +17,7 @@ defmodule Notox.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    get "/notes", NoteController, :index
```

Comment:
Actually, I would do it even if I wasn't reasonably certain. It's the opposite. I only use `get`, `patch`, etc when I'm reasonably certain I _won't_ be adding additional routes for the path. One example of that is novelty routes, where you have a `get` to `sign_up`, but the rest of the actions would be to `/users`.